src/python/dxpy/scripts/dx_build_app.py

```diff
@@ -472,6 +472,8 @@ def _verify_app_writable(app_name):
     except dxpy.exceptions.DXAPIError as e:
         if e.name == 'ResourceNotFound':
             app_name_already_exists = False
+        elif type(e) == dxpy.exceptions.PermissionDenied:
```

Comment:
This should be in a similar format to the above e.g. `e.name == "PermissionDenied"`